mix.exs

```diff
@@ -41,7 +41,10 @@ defmodule Thrift.Mixfile do
      package: package(),
 
      # Dialyzer
-     dialyzer: [plt_add_deps: :transitive, plt_add_apps: [:ex_unit, :mix, :public_key, :ssl]],
+     dialyzer: [
+       plt_add_deps: :transitive,
+       plt_add_apps: [:mix],
```

Comment:
You had added `:ex_unit`, and then I added `:public_key` and `:ssl` when I was adding specs for some of the ranch types. Those three no longer appear to be explicitly needed now, perhaps due to this dialyzer upgrade. I went through a few clean builds to be sure.

`:mix` is still necessary.